packages/components/pie-button/src/index.ts

```diff
@@ -1,10 +1,27 @@
-import { LitElement, html } from 'lit';
+import { LitElement, html, css } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 
 @customElement('pie-button')
 export class PieButton extends LitElement {
     @property() actionType = 'submit';
 
+    static styles = [css`
+      button {
+        background-color: var(--dt-color-interactive-brand);
+        border-radius: var(--dt-radius-rounded-e);
+        border: none;
+        color: var(--dt-color-content-interactive-primary);
+        font-size: 20px; // A future ticket will pull in a helper for font size & line height.
+        line-height: 1.4;
+        font-family: var(--dt-font-interactive-m-family);
+        font-weight: var(--dt-font-interactive-m-weight);
+        padding: auto;
```

Comment:
i think `padding: auto;` is not a valid CSS value

<img width="491" alt="Screenshot 2023-01-24 at 16 47 16" src="https://user-images.githubusercontent.com/28605803/214340466-c23f2676-a98a-4468-93ac-19e1465c0fe4.png">

Reply:
Hmm so I guess if we don't set vertical padding at all then it should still work?